README.md

```diff
@@ -1,20 +1,50 @@
 
-RT & BT activist of BeS community.
+Red Team & Blue Team activist of BeS community.
 
-## Features of oah-bes-vm?
+### Red Team (RT)
+A group of people authorized and organized to emulate a potential adversary's attack or exploitation capabilities against an enterprise's security posture.
+
+### Blue Team (BT)
+The group responsible for defending an enterprise's use of information systems by maintaining its security posture against a group of mock attackers (i.e., the Red Team).
+
+### Usage scenarios:
+1. **Security Analyst**: Security analysts are experts who identify and fix problems within security systems by analyzing risks, vulnerabilities, threats and incidents. They perform a ton of security analyses to recommend solutions for preventing cyberattacks. RT & BT activist / Security Analyst of BeS community can use oah-bes-vm to work on a specific open-source security project.
 
-1. Offers bundles of BeS tools that would enable users to fast track their Red Teaming / Blue Teaming activities by significantly reducing effort spent on setting up respective environments.
-2. It helps to isolate the installation of BeS projects in a specific environment.
-3. According to the needs, user can configure oah-bes-vm during installation.
+2. **BeSLab Manager**: oah-bes-vm helps BeSLab Manager & Security Analyst to analyse and do RT & BT activities of open-source security projects. After analysing multiple open-source security projects using multiple static analysis tools (like SonarQube, Codeql, SBOM etc) by BeSLab Manager, Security Analyst can use oah-bes-vm to setup an isolated RT/BT environment for each spesific open-source security project to do RT & BT activities.
+([BeSLab Documentation](https://be-secure.github.io/Be-Secure/bes-beslab-details/))
 
+## Features of oah-bes-vm?
+1. oah-bes-vm allow user to setup RT & BT environment inside a ubuntu based virtual box.
+2. User can install, remove, update, reset & validate RT & BT environment for a specific open-source security project as per requirments.
+   - ### Install an environment
+            bes install -env <environment name> -V <version>
+     ![install environment](docs/img/env_install.gif)
+     
+   - ### Validate an environment
+            bes validate -env <environment name>
+     ![validate environment](docs/img/env_validate.gif)
+
+   - ### Update an environment
+            bes update -env <environment name>
+     ![update environment](docs/img/env_update.gif)
+
+   - ### Reset an environment
+            bes reset -env <environment name>
+     ![reset environment](docs/img/env_reset.gif)
+
+   - ### Uninstall an environment
+            bes uninstall -env <environment name>
+     ![uninstall environment](docs/img/env_remove.gif)
+
+    For more information check [BeSman documentation](https://be-secure.github.io/Be-Secure/bes-besman-details/).
 ## List of tools
 ### BeS Tools
-1. BeSman
+1. [BeSman](https://be-secure.github.io/Be-Secure/bes-besman-details/)
```

Comment:
Can you give some overview on List of tools and why they are required in oah-bes-vm?